packages/desktop-client/src/components/reports/Overview.js

```diff
@@ -325,12 +327,48 @@ function CategorySpendingCard() {
   );
 }
 
+function SankeyCard({ categories }) {
+  const end = monthUtils.currentMonth();
+  const start = monthUtils.subMonths(end, 5);
+
+  const params = useMemo(
+    () => sankeySpreadsheet(start, end, categories),
+    [start, end, categories],
+  );
+  const data = useReport('sankey', params);
+
+  return (
+    <Card flex={1} to="/reports/sankey">
+      <View style={{ flexDirection: 'row', padding: 20 }}>
+        <View style={{ flex: 1 }}>
+          <Block
+            style={{ ...styles.mediumText, fontWeight: 500, marginBottom: 5 }}
+            role="heading"
+          >
+            Sankey
+          </Block>
+          <DateRange start={start} end={end} />
+        </View>
+      </View>
+      <View style={{ flex: 1 }}>
+        {data ? (
+          <SankeyGraph data={data} compact={true} />
+        ) : (
+          <LoadingIndicator />
+        )}
+      </View>
+    </Card>
+  );
+}
+
 export default function Overview() {
   let categorySpendingReportFeatureFlag = useFeatureFlag(
     'categorySpendingReport',
   );
+  let sankeyFeatureFlag = useFeatureFlag('sankeyReport');
 
   let accounts = useSelector(state => state.queries.accounts);
+  let categories = useSelector(state => state.queries.categories.grouped);
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏you might want to use `useCategories` here. It fetches the categories if they have not been fetched already. Whereas your current implementation just returns the cache.